timed/settings.py

```diff
@@ -158,12 +143,12 @@ def trueish(value):
     ),
     'DEFAULT_PARSER_CLASSES': (
         'rest_framework_json_api.parsers.JSONParser',
+        'rest_framework.parsers.JSONParser',
         'rest_framework.parsers.FormParser',
         'rest_framework.parsers.MultiPartParser'
     ),
     'DEFAULT_PERMISSION_CLASSES': (
         'rest_framework.permissions.IsAuthenticated',
-        'rest_framework.permissions.DjangoModelPermissions',
```

Comment:
At the moment we don't need any complex permission, since we only have readonly models or models which can only be edited / deleted / created by their owner. It is not in sight that we need more complex permission but if we do, we can still add it back.

The removal of this permission class make testing much easier, since we don't have to add all the permissions to the users.